test/index.js

```diff
@@ -20,3 +20,17 @@ test('chains multiple reducers into a single reducer', t => {
   t.deepEqual(reducerAddMult({ A: 1, B: 2 }, 3), { A: 12, B: 2 });
   t.deepEqual(reducerMultAdd({ A: 1, B: 2 }, 3), { A: 6, B: 2 });
 });
+
+test('should support additional arguments', t => {
```

Comment:
Please use `supports` instead of `should support`.

Reply:
fixed and rebased